### PR TITLE
Switch curator container image to use bobrik/curator:5.4.0

### PR DESCRIPTION
**Pull request by jar361**
Move configmap values to values.yaml

### CODE REVIEW

**Comment by jar361** [comment]
Waiting to merge https://github.com/samsung-cnct/chart-curator/pull/13

File: curator/values.yaml [hunk 1]
```diff
@@ -3,6 +3,53 @@
 # Declare name/value pairs to be passed into your templates.
 # name: value
 
-image: quay.io/samsung_cnct/curator-container
+image: bobrik/curator
```
Comment (DStorck) [inline curator/values.yaml]: We prefer to use our own image for CI purposes

Comment (rendhalver) [reply curator/values.yaml]: We switched to that image because it has more versions available.
Your image has only one tag and it doesn't support ES 6.

**Comment by alika** [comment]
@jar361 We have incorporated your configmap changes in [this PR](https://github.com/samsung-cnct/chart-curator/pull/17). They are much more flexible. To accommodate your curator image preference you can run the following if you have helm and the helm registry plugin installed:

```
helm registry upgrade quay.io/samsung_cnct/curator@0.0.1-47 \
  -- ${RELEASE} --install --namespace ${NAMESPACE} \
  --set image.name=bobrik/curator --set image.tag=5.4.0
```

If you do not feel like setting up the helm registry plugin you can use docker and run the following:

```
docker run --rm \
  -v ${KUBECONFIG}:/root/.kube/config \
  quay.io/samsung_cnct/helm-registry-agent:v0.7.4-helm_2.7 \
  helm registry upgrade quay.io/samsung_cnct/curator@0.0.1-47 \
    -- ${RELEASE} --install --namespace ${NAMESPACE} \
    --set image.name=bobrik/curator --set image.tag=5.4.0
```

**Pull request closed without merging**